model/Cargo.toml

```diff
@@ -21,10 +21,18 @@ serde-mappable-seq = { default-features = false, version = "0.1" }
 serde-value = { default-features = false, version = "0.7" }
 serde_repr = { default-features = false, version = "0.1" }
 tracing = { default-features = false, version = "0.1" }
+thiserror = "1.0.12"
```

Comment:
Just a small nit this should probably be removed to match with the rest of the crates